### DISCUSSION TITLE
New User Thread

### DISCUSSION THREAD

**Discussion by majestrate** [majestrate/XD#108] [General]
This thread is for new user questions and chatter.

**Comment by p060477** [comment]
Hi, there is no way to put a torrent url in yr webui...it only accepts magnets urls...why...??
is it a bug...?...if yes should i have to open an issue...??

**Comment by p060477** [comment]
have you see this...??...:
https://github.com/majestrate/XD/issues/110

**Reply by majestrate** [reply]
yes, i check my email every morning.

**Comment by p060477** [comment]
and how to be able to finally use torrent urls in webui as yr sw state ...??

**Comment by cramik** [comment]
Does the lokinet DNS option need to be set? There's no guide

**Reply by majestrate** [reply]
the lokinet dns resolver can be set manually using the environmental variable `LOKINET_DNS`. i have not thoroughly tested that feature so please open any bug reports if (when) you encounter problems with it.
